### ISSUE TITLE
Support api resources without collection operation in EntrypointNormalizer

### ISSUE THREAD

**Issue by BacLuc** [ecamp/ecamp3#2347]
Leider werden mit dem Hal/EntrypointNormalizer Ressourcen ohne collection operations ignoriert.

https://github.com/api-platform/core/blob/8cc452c1faa29aed69a2cd43044859c388861147/src/Hal/Serializer/EntrypointNormalizer.php#L71

Könnte sein, weil der IriConverter das (noch) nicht unterstützt.
Wir mussten das jetzt schon 2 mal mit fake collection enpoints umgehen, wäre gut wenn wir das lösen könnten.
(Wahrscheinlich am besten mit PR bei Api Platform).

**Comment by carlobeltrame** [comment]
Ja, oder nicht mehr auf den IriConverter setzen bzw. bei uns erweitern? Der scheint mir eher eine high-level Utility zu sein, und wenn der für uns nicht mehr ausreicht...
